tests/cloudpickle_test.py

```diff
@@ -140,6 +140,16 @@ def test_save_unsupported(self):
 
         assert "Cannot pickle objects of type" in str(excinfo.value)
 
+    def test_pickling_sys_stdin(self):
+        self.assertRaisesRegexp(pickle.PicklingError,
```

Comment:
Since PySpark supports Python 2.6, we should either use a different testing setup:

```
with pytest.raises(pickle.PicklingError) as excinfo:
    cloudpickle.dumps(sys.stdin)
assert "Cannot pickle standard input" in excinfo.value
```

or unittest2 needs to be installed for Python 2.6.